controllers/etcd_custodian_controller.go

```diff
@@ -168,6 +176,89 @@ func (ec *EtcdCustodian) updateEtcdStatus(ctx context.Context, logger logr.Logge
 	})
 }
 
+func calculatePDBminAvailable(etcd *druidv1alpha1.Etcd) int {
+	// do not enable in single mode
+	if etcd.Spec.Replicas < 2 {
+		return 0
+	}
+
+	// cluster is in bootstrap
+	if etcd.Status.ClusterSize == nil {
+		return 0
```

Comment:
We should not return `0` here because Custodian can accidentally reset `minAvailable: 0` when bootstrapping is happening and the value should actually be `int(etcd.Spec.Replicas)`. I suggest to do nothing in case of bootstrapping and leave it up to the `etcd_controller` to set the correct value.